### ISSUE TITLE
[A11Y] Contrast Issues

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/polls#4784]
### Elements found:

The subtitle of the private poll is not accessible
<img width="487" height="88" alt="Image" src="https://github.com/user-attachments/assets/ac2ab5f4-0ffd-4590-bdf6-be1fba9c5ee7" />

The yellow 0 is not accessible
<img width="198" height="244" alt="Image" src="https://github.com/user-attachments/assets/eeaabdfb-8a61-4a73-849c-b0499a2e7282" />

### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on Github _(I've searched it)_.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

Certain text in this app doesn't pass the WCAG AA standard

### What did you expect, how polls should behave instead?

All the text in the app should pass WCAG AA standard

### What steps does it need to replay this bug?

1. Open mentioned parts of app, use extension like WCAG Color contrast checker

### Affected polls version

main

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

First time installation

### Can you rule out that any extension you use is involved in the issue?

- [ ] I have checked all browser extension

### Which browser did you use, when experiencing the bug?

- [x] Firefox
- [ ] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Add your browser log here

```json

```

### NC version

Nextcloud 33 (Hub 26 Winter)

### PHP engine version

PHP 8.1

### Database engine

MySQL

### Which user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```json

```

### Configuration report

```shell

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```
